app/src/main/java/org/mozilla/fenix/tabstray/TabsTrayInteractor.kt

```diff
@@ -54,6 +45,51 @@ interface TabsTrayInteractor : SyncedTabsInteractor {
         targetId: String?,
         placeAfter: Boolean,
     )
+
+    /**
+     * TabTray's Floating Action Button clicked.
```

Comment:
```suggestion
     * Invoked when the tab tray's floating action button is clicked.
```

Reply:
Let's also go through the other comments and come up with a common way that we want to phrase the comments.